### ISSUE TITLE
Error: Cannot find module ' react-hot-loader/babel' While deploying to heroku

### ISSUE THREAD

**Issue by Zcery** [Bikranshu/express-react-boilerplate#49]

**Comment by Bikranshu** [comment]
Have you tried **heroku config:set NPM_CONFIG_PRODUCTION=false**

**Comment by Zcery** [comment]
Yes, I did that but there were few more issues with port binding as this command deploy the complete app including devdependencies which also increases the size from 34 MB to 88MB

**Comment by TrustDev** [comment]
I updated package.json. You can change "scripts" option as belows.

`  "scripts": {
    "start": "babel-node server/app.js",
    "clean": "rimraf node_modules && rimraf dist && npm install",
    "webpack:dev": "cross-env NODE_ENV=development webpack --config ./webpack/webpack.config.dev.js",
    "webpack:prod": "cross-env NODE_ENV=production NODE_OPTIONS=--openssl-legacy-provider webpack --config ./webpack/webpack.config.prod.js",
    "build:dev": "rimraf dist && npm run webpack:dev && npm run start",
    "build": "rimraf dist && npm run webpack:prod && npm run start",
    "migrate": "knex migrate:latest",
    "migrate:make": "knex migrate:make",
    "migrate:rollback": "knex migrate:rollback",
    "lint": "eslint client server; exit 0"
  },`
